luigi/contrib/gcs.py

```diff
@@ -231,7 +234,7 @@ def remove(self, path, recursive=True):
                 raise InvalidDeleteException(
                     'Path {} is a directory. Must use recursive delete'.format(path))
 
-            req = http.BatchHttpRequest()
+            req = http.BatchHttpRequest(batch_uri=GCS_BATCH_URI)
```

Comment:
So google api client doesn't set this as default? It's a bit surprising.

Reply:
The default still goes to the global batch uri (https://www.googleapis.com/batch) - now there's a different batch endpoint for each api (but BatchHttpRequest is still generic to request type) so you need to set it to the uri for whatever api you are using